catalog/video/video-mpegts-stream/src/test/java/org/codice/alliance/video/stream/mpegts/rollover/ByteCountRolloverConditionTest.java

```diff
@@ -59,16 +59,16 @@ public void testCurrentEqualToThreshold() {
     @Test
     public void testCurrentGreaterThanThreshold() {
 
-        when(packetBuffer.getByteCount()).thenReturn(THRESHOLD + 1);
+        when(packetBuffer.getByteCount()).thenReturn(getThresholdAsBytes() + 1);
 
         assertThat(condition.isRolloverReady(packetBuffer), is(true));
 
     }
 
     @Test
     public void testSetter() {
-        condition.setByteCountThreshold(THRESHOLD - 1);
-        assertThat(condition.getByteCountThreshold(), is(THRESHOLD - 1));
+        condition.setMegabyteCountThreshold(MEGABYTE_THRESHOLD - 1);
```

Comment:
❗️ We probably should test the setter and constructor validation edge cases (below min and over max). We could also just keep the setter and remove the constructor and not have to test the latter.